pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -104,39 +104,28 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 	return wc, nil
 }
 
-const (
-	// GMSASpecContainerAnnotationKey is the container annotation where we store the contents of the GMSA credential spec to use.
-	GMSASpecContainerAnnotationKey = "container.alpha.windows.kubernetes.io/gmsa-credential-spec"
-	// gMSAContainerSpecPodAnnotationKeySuffix is the suffix of the pod annotation where the GMSA webhook admission controller
-	// stores the contents of the GMSA credential spec for a given container (the full annotation being the container's name
-	// with this suffix appended).
-	gMSAContainerSpecPodAnnotationKeySuffix = "." + GMSASpecContainerAnnotationKey
-	// gMSAPodSpecPodAnnotationKey is the pod annotation where the GMSA webhook admission controller stores the contents of the GMSA
-	// credential spec to use for containers that do not have their own specific GMSA cred spec set via a
-	// gMSAContainerSpecPodAnnotationKeySuffix annotation as explained above
-	gMSAPodSpecPodAnnotationKey = "pod.alpha.windows.kubernetes.io/gmsa-credential-spec"
-)
-
-// determineEffectiveSecurityContext determines the effective GMSA credential spec and, if any, copies it to the container's
-// GMSASpecContainerAnnotationKey annotation.
+// determineEffectiveSecurityContext determines the effective GMSA credential spec and, if any, copies it
+// to the config's `.Windows.SecurityContext.GmsaCredentialSpec`.
 func determineEffectiveSecurityContext(config *runtimeapi.ContainerConfig, container *v1.Container, pod *v1.Pod) {
```

Comment:
Will it be possible to get rid of the `determineEffectiveSecurityContext` here and transfer the logic below to `securitycontext.DetermineEffectiveSecurityContext` above?